### PR TITLE
Terrains - Khe Sanh map port

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Porten Khe Sanh Map

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### What needs to be tested? 
1. [ ] Marker Placement
2. [ ] Initial HQ Placement
3. [ ] Proper spawns

### What needs to be overhauled?
1. [ ] Pictures
2. [ ] String Texts

********************************************************
Notes:

### CODE REVIEW

File: Map-Templates/Antistasi-Khe-Sanh.vn_khe_sanh/Map/vn_khe_sanhInfo.sqf [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    License: MIT license
+*/
+#include "..\Includes\common.inc"
+
+params["_filename", ""];
+
+switch (_filename) do {
+	case "population": {
+		//To improve Performance, reduces pop from 13972 to 4850
```
Comment (Lazejun) [inline Map-Templates/Antistasi-Khe-Sanh.vn_khe_sanh/Map/vn_khe_sanhInfo.sqf:29]: This comment is taken from Livonia's Map Info.

Comment (Bob-Murphy) [reply Map-Templates/Antistasi-Khe-Sanh.vn_khe_sanh/Map/vn_khe_sanhInfo.sqf:29]: True.
I more slapped it together from what I had before the system change.
Need to check it on a quite minute but wanted to have the stuff already properly put somewhere.

**Comment by killerswin2** [comment]
When you have a moment Bob, you need to merge in unstable to remove all the extra files that you added.

**Comment by Bob-Murphy** [comment]
outdated and replaced by #2224

**Pull request closed without merging**